PasswordRulesSharp.Cli/Commands/GenerateCommand.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.ComponentModel;
+
+using Spectre.Console;
+using Spectre.Console.Cli;
+
+using Rule = PasswordRulesSharp.Rules.Rule;
+
+namespace PasswordRulesSharp.Cli.Commands;
+
+public class GenerateCommand : Command<GenerateCommand.Settings>
+{
+    public class Settings : CommandSettings
+    {
+        [CommandOption("--rule")]

```

Comment:
Wait, why an option? Isn't this always required? Why not an argument so I don't need to remember the option name

Reply:
Makes sense.